### PR TITLE
OnDataResumed Notification

### PR DESCRIPTION
**Pull request by JackLivio**
This proposal creates a new RPC notification that relays app resumption info from SDL Core to the mobile app after registration. The notification is named `OnDataResumed`.

### CODE REVIEW

**Comment by JackLivio** [comment]
@pelsila Ready for review

File: proposals/nnnn-on-data-resumed.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
+# OnDataResumed Notification
+
+* Proposal: [SDL-NNNN](NNNN-on-data-resumed-notification.md)
+* Author: [JacLivio](https://github.com/JackLivio)
```
Comment (pelsila) [inline proposals/nnnn-on-data-resumed.md:4]:
```suggestion
* Author: [JackLivio](https://github.com/JackLivio)
```

**Review by pelsila — CHANGES_REQUESTED** [review]
@JackLivio Just one typo.  Let me know if you have any questions.  Thank you

**Comment by JackLivio** [comment]
@pelsila Updated

**Review by pelsila — APPROVED** [review]

**Pull request merged**